.github/workflows.src/tests.inc.yml

```diff
@@ -12,7 +12,7 @@
     - name: Set up Python
       uses: actions/setup-python@v4
       with:
-        python-version: '3.10.12'
```

Comment:
@elprans this probably means we should issue future (nightly) releases in 3.11 too, right?

Reply:
We already do. 3.x as well

Reply:
https://github.com/edgedb/edgedb-pkg/blob/4cfef6561b97bf54de134d3b9e81c70bdc7197ba/edgedbpkg/edgedb/__init__.py#L64-L73